.github/workflows/ci.yml

```diff
@@ -27,43 +27,11 @@ name: CI
 # https://docs.github.com/en/actions/hosting-your-own-runners/using-self-hosted-runners-in-a-workflow
 
 on:
-  push:
-    branches:
-      - main
-    paths:
-      - ".github/workflows/ci.yml"
-      - ".github/workflows/setup.yml"
-      - ".bazelrc"
-      - ".bazelversion"
-      - "WORKSPACE"
-      - "CMakeLists.txt"
-      - "build_tools/**"
-      - "compiler/**"
-      - "llvm-external-projects/**"
-      - "runtime/**"
-      - "samples/**"
-      - "tests/**"
-      - "third_party/**"
-      - "tools/**"
+  workflow_dispatch:
   pull_request:
+  push:
     branches:
       - main
-    paths:
-      - ".github/workflows/ci.yml"
-      - ".github/workflows/setup.yml"
```

Comment:
Could keep the conditions on `push` but I'd rather have them consistent with the conditions in `pull_request`.